src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/SignatureHelp/SignatureHelpEndpointTest.cs

```diff
@@ -82,27 +72,11 @@ private async Task VerifySignatureHelpAsync(string input, params string[] signat
             // Arrange
             TestFileMarkupParser.GetPositionAndSpans(input, out var output, out int cursorPosition, out ImmutableArray<TextSpan> spans);
             var codeDocument = CreateCodeDocument(output);
-            var csharpSourceText = codeDocument.GetCSharpSourceText();
-            var csharpDocumentUri = new Uri("C:/path/to/file.razor__virtual.g.cs");
-            var serverCapabilities = new ServerCapabilities()
-            {
-                DocumentHighlightProvider = true
```

Comment:
So this was wrong.. but it turns out we don't even need to set the server capabilities for these.. which I guess is not surprising since we're directly calling `HandleRequestAsync` on the test server.

Reply:
Oops, this was probably me - thanks for the catch!